grunt_tasks/webpack.config.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
             // JSON files
             {
                 test: /\.json$/,
-                include: loaderPaths,
+                include: [...loaderPaths, ...loaderPathsNodeModules],
```

Comment:
v4 is now what our docs prescribe for installation. (We are actually already using ES2015 features in our build code besides this).